src/main/java/seedu/address/flashcard/Answer.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.flashcard;
+
+/**
+ * Represents an Answer in a flashcard.
+ */
+public class Answer {
+    final private String answer;
+
+    public Answer(String answer) {
+        this.answer = answer;
+    }
+
+    /**
+     * Checks the given userAnswer with the correct answer.
+     * This is done by strictly comparing lower case string equality.
+     * @param userAnswer the user's answer.
+     * @return true if the user's answer is equal to the actual answer.
+     */
+    public boolean checkAnswer(String userAnswer) {
+        String lowerCaseActualAnswer = answer.toLowerCase();
+        String lowerCaseUserAnswer = userAnswer.toLowerCase();
+        return lowerCaseActualAnswer.equals(lowerCaseUserAnswer);
+    }
+
```

Comment:
Looks good! Just wondering whether it would be better if we made userAnswer an Answer type also, that way we can potentially do more things with it in the future. 

If userAnswer was an Answer type, we can just override `.equals()` as a way of comparison to check if userAnswer is indeed Answer.